### ISSUE TITLE
Feature request: Add volume to ch-companion

### ISSUE THREAD

**Issue by RashedKhalfan** [matteoantoci/ch-companion#4]
I think in addition to volatility  and max change I think volume will make more sense since sometimes we end up with some coins that have a low volume which makes it hard to sell

**Comment by jovancosta-zz** [comment]
Yep @matteoantoci I totally agree with this suggestion. Could you please add min volume on the parameters?

**Comment by Handiman918** [comment]
Yes, @matteoantoci & RashedKhalfan, is min Volume filter possible?
